content/600-about/200-prisma-docs/20-style-guide/index.mdx

```diff
@@ -366,6 +366,14 @@ npm install prisma
 
 Always use `npm` commands instead of `yarn`.
 
+### Package versioning
+
+Include the major version of a package in the installation step. This ensures compatibility and reduces the maintenance overhead when a package introduces a breaking change:
+
+```terminal
+npm install --save-dev typescript@4
+```
+
```

Comment:
Maybe include another paragraph about what the pinned version looks like inside of `package.json`.